src/pages/Products/Products.js

```diff
@@ -43,17 +60,25 @@ export function Products() {
 
   useEffect(() => {
     if (!searchValue) {
-      searchParams.delete("search");
+      searchParams.set("page", currentPage);
+      setSearchParams(searchParams);
+      loadProducts();
+      return;
+    }
+    if (searchValue) {
```

Comment:
combine it into the single if/else statement

if (searchValue) {

} else { }